### ISSUE TITLE
Show the double click function on the UI

### ISSUE THREAD

**Issue by honza-zidek** [tchiotludo/akhq#1166]
When I see the list of all topics and I want to see details of a topic, I have to click on the magnifier icon on the right.
It would make much more sense if simply the whole topic line were clickable to open the topic details.

**Comment by tchiotludo** [comment]
There is an hidden (and magic) fonction, double click on the rows :tada:

**Comment by tchiotludo** [comment]
I know it's cryptic and not common, but enable on single click prevent to copy paste a name for example. 
It's why I decided to good to double click that only power user (like you now :smile:) will know. 
I don't have any other option if I want to keep the copy paste as I know

**Comment by honza-zidek** [comment]
> but enable on single click prevent to copy paste a name for example
sounds like a valid point.

Can you think about a way to tell users that this magic function is there? E.g. when I hover the mouse over the magnifier, there is a tooltip "Details", you may enhance the text to "Details (also double-click wherever on the whole row)", or something like this. And/or tooltip over the whole line may tell "Double-click to view details".

**Comment by tchiotludo** [comment]
I reopen to gather idea on that subject

**Comment by tristanessquare** [comment]
If you want to enable users to copy for example the name of the topic then you could place a little copy button/icon besides the name. In other UIs that is a quite common solution.

**Comment by Dugong42** [comment]
I agree, it's a shame for such a basic function to be hidden behind a non-standard behaviour.
I think there would be far less confusion and better accessibility if the topic names were simply clickable as links.

It's always possible to copy a link by draging from its side, so I don't think the double-click workaround is really necessary.

**Comment by Dugong42** [comment]
As a suggestion, I think it would also be interesting to enlarge the clickable area of the delete/config buttons. The buttons are really small so it does not favour accessibility.
